modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/change/tracking/spi/reference/DDMFormInstanceRecordVersionTableReferenceDefinition.java

```diff
@@ -29,6 +30,11 @@ public class DDMFormInstanceRecordVersionTableReferenceDefinition
 	public void defineChildTableReferences(
 		ChildTableReferenceInfoBuilder<DDMFormInstanceRecordVersionTable>
 			childTableReferenceInfoBuilder) {
+
+		childTableReferenceInfoBuilder.assetEntryReference(
+			DDMFormInstanceRecordVersionTable.INSTANCE.
+				formInstanceRecordVersionId,
+			DDMFormInstanceRecord.class);
```

Comment:
Not needed. AssetEntry is not created when a DDMFormInstanceRecordVersion is added.